static/app/components/replays/breadcrumbs/replayBreadcrumbOverview.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import range from 'lodash/range';
+import moment from 'moment';
+
+import {transformCrumbs} from 'sentry/components/events/interfaces/breadcrumbs/utils';
+import space from 'sentry/styles/space';
+import {Crumb, RawCrumb} from 'sentry/types/breadcrumbs';
+
+import {countColumns, formatTime, getCrumbsByColumn} from '../utils';
+
+import StackedContent from './stackedContent';
+
+const EVENT_STICK_MARKER_WIDTH = 2;
+
+interface Props {
+  data: {
+    values: Array<RawCrumb>;
+  };
+}
+
+type LineStyle = 'dotted' | 'solid' | 'none';
+
+function ReplayBreadcrumbOverview({data}: Props) {
+  const transformedCrumbs = transformCrumbs(data.values);
+
+  return (
+    <StackedContent>
+      {({width}) => (
+        <React.Fragment>
+          <Ticks
+            crumbs={transformedCrumbs}
+            width={width}
+            minWidth={20}
+            lineStyle="dotted"
+          />
+          <Ticks
+            crumbs={transformedCrumbs}
+            width={width}
+            showTimestamp
+            minWidth={50}
+            lineStyle="solid"
+          />
+          <Events crumbs={transformedCrumbs} width={width} />
+        </React.Fragment>
+      )}
+    </StackedContent>
+  );
+}
+
+function Ticks({
+  crumbs,
+  width,
+  minWidth = 50,
+  showTimestamp,
+  lineStyle = 'solid',
+}: {
+  crumbs: Crumb[];
+  lineStyle: LineStyle;
+  width: number;
+  minWidth?: number;
+  showTimestamp?: boolean;
+}) {
+  const startTime = crumbs[0]?.timestamp;
+  const endTime = crumbs[crumbs.length - 1]?.timestamp;
+
+  const startMilliSeconds = moment(startTime).valueOf();
+  const endMilliSeconds = moment(endTime).valueOf();
+  const duration = endMilliSeconds - startMilliSeconds;
+
+  const {timespan, cols, remaining} = countColumns(duration, width, minWidth);
+
+  return (
+    <TimelineMarkerList totalColumns={cols} remainder={remaining}>
+      {range(0, cols).map((_, i) => (
```

Comment:
What about

```suggestion
      {[...Array(cols)].map((_, i) => (
```

tbh, `range()` is probably more clear.